.github/workflows/release.yml

```diff
@@ -8,60 +8,39 @@ on:
 
 jobs:
   windows:
-    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        arch: ["x86-64-bmi2", "x86-64-modern", "x86-64"]
+    runs-on: windows-latest
```

Comment:
I prefer to use stable identifiers (like windows-2019) to avoid scenarios where CI can suddenly fail without code changes, e.g. when github updates what is `latest`.

Reply:
I've updated the workflow.